shared/source/os_interface/linux/drm_neo.cpp

```diff
@@ -488,6 +489,19 @@ std::string Drm::generateUUID() {
     return std::string(buffer, 36);
 }
 
+PhysicalDevicePciBusInfo Drm::getPciBusInfo() const {
+    PhysicalDevicePciBusInfo pciBusInfo;
+
+    if (hwDeviceId) {
+        pciBusInfo.pciDomain = 0;
+        if (std::sscanf(hwDeviceId->getPciPath(), "%02x:%02x.%01x", &(pciBusInfo.pciBus), &(pciBusInfo.pciDevice), &(pciBusInfo.pciFunction)) != 3) {
```

Comment:
please add const for "3" to avoid magic numbers

Reply:
The constant value has been added.